server/src/modules/runs/runs.service.ts

```diff
@@ -64,4 +65,18 @@ export class RunsService {
 
         return new PaginatedResponseDto(RunDto, dbResponse);
     }
+
+    async deleteStoredMapRuns(mapID: number): Promise<void> {
+        const where: Prisma.RunWhereInput = {
+            mapID
+        };
+        const dbResponse = await this.runRepo.getAllRuns(where);
+        const runs = dbResponse.flatMap<Run>((resp) => resp[0]);
+
+        const runFileDeletes = runs.map((run) => {
+            this.fileCloudService.deleteFileCloud(`runs/${run.id}`);
+        });
```

Comment:
Unnecessary arrow func braces

Reply:
Again a weird assignment, I was just promise.all the mapping immediately. Not a big deal though.